techniques/general/GXX_motionactuationfailure.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
+	<head>
+		<title>Failure due to inability to deactivate motion actuation</title>
+		<link rel="stylesheet" type="text/css" href="../../css/editors.css"/>
+	</head>
+	<body>
+		<h1>Failure due to inability to deactivate motion actuation</h1>
+		<section id="meta">
+			<h2>Metadata</h2>
+			<p id="id"></p>
+			<p id="technology"></p>
+			<p id="type"></p>
+		</section>
+		<section id="applicability">
+			<h2>Applicability</h2>
+			<p>Content using technologies that can support the detection of device or user motion such as shaking or tilting and use these motions as a means of input.</p>
+		</section>
+		<section id="description">
+			<h2>Description</h2>
+			<p>This describes the failure condition that results when motion actuation can not be deactivated.  People who may accidentally activate sensors due to tremors or other motor impairments need the ability to turn off motion actuation to prevent such accidental triggering of functions.  </p>
+			<p>The type of motion covered by the <a href="../../guidelines/sc/21/motion-actuation.html">Motion Actuation Success Criterion</a> does not relate to the movement of users through space as registered by geolocation sensors or beacons, or events observed by the device other than intentional gesturing by the user. It also does not cover indirect motion associated with operating a keyboard, pointer, or assistive technology.</p>
+			
+		</section>
+		<section id="examples">
+			<h2>Examples</h2>
+	
+			<section class="example">
+				<h3>Motion Activated Slider</h3>
+				<p>A slider which uses tilting motion to increase or decrease the value of an input, with no mechanism to deactivate the motion detection.</p>
+				<p class="working-example"><a href="../../working-examples/device-motion-sensor-input-failure/">Working example of a motion activated slider that can not be deactivated</a></p>
+			</section>
+		</section>
+		<section id="tests">
+			<h2>Tests</h2>
+			<section class="test-procedure">
+				<h3>Procedure</h3>
+				<p>For each function that is triggered by a motion sensor:</p>

```

Comment:
@awkawk I updated the procedure to account for the exceptions in the SC text, if it impacts what you were going to look at for the other failures?